README.md

```diff
@@ -17,19 +17,48 @@ sudo bash ./install.sh
 ```
 
 ### Getting Started
-```
-replicated channel ls --app my-app-slug --token e8d7ce8e3d3278a8b1255237e6310069
+
+You can use the replicated CLI command (i.e. `$ replicated --help`). 
+
+#### (Pre-requirement) Config the CLI to have access to the Replicated Vendor portal 
+
+To use the CLI you need a user API Token to connect. You can create one by clicking on `New User API Token` in your 
+[Account Settings](https://vendor.replicated.com/account-settings). 
+
+Then, you can set the following environment variable to avoid passing it as an argument to each command
+
+```shell
+$ export REPLICATED_API_TOKEN=<Your new User API Token>
 ```
 
-Set the following env vars to avoid passing them as arguments to each command.
-* REPLICATED_APP - either an app slug or app ID
-* REPLICATED_API_TOKEN
+#### Now, let's check an example
 
-Then the above command would be simply
+We will check all Applications available as its distribution channels:
```

Comment:
I think I see what you mean here.  This may be a better change?

```suggestion
We will check all Applications available and their distribution channels:
```